worker/pkg/zip/zip_test.go

```diff
@@ -0,0 +1,68 @@
+package zip
+
+import (
+	"archive/zip"
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Buffer implements io.ReadAtCloser and io.WriteCloser
+type Buffer struct {
+	bytes.Buffer
+}
+
+func (*Buffer) Close() error {
+	return nil
+}
```

Comment:
@rot1024 
Thanks for reviewing my PR. Basically, I accepted your suggested change. 
Additionally, I changed `pkg/zip` for its testing way so that is doesn't need to generate files and zip files dynamically.